x/claims/keeper/hooks_test.go

```diff
@@ -341,9 +337,7 @@ func (suite *KeeperTestSuite) TestAfterEVMStateTransition() {
 				expBalance := suite.app.BankKeeper.GetBalance(suite.ctx, addr, params.ClaimsDenom)
 
 				coins := sdk.Coins{sdk.NewCoin(params.ClaimsDenom, sdk.NewInt(250))}
-				err := suite.app.BankKeeper.MintCoins(suite.ctx, inflationtypes.ModuleName, coins)
-				suite.Require().NoError(err)
-				err = suite.app.BankKeeper.SendCoinsFromModuleToModule(suite.ctx, inflationtypes.ModuleName, types.ModuleName, coins)
+				err := testutil.FundModuleAccount(suite.app.BankKeeper, suite.ctx, types.ModuleName, coins)
```

Comment:
Generally prefer refactors like this to be made in their own PRs to reduce review noise. Applies to all `FundModuleAccount` changes